### PR TITLE
Add Databases to Datasync from Integration to Training

### PR DESCRIPTION
**Pull request by camdesgov**
We need to sync these databases for the Training environment
to function correctly.

### CODE REVIEW

File: hieradata_aws/class/training/db_admin.yaml [hunk 1]
```diff
@@ -43,3 +43,201 @@ govuk_env_sync::tasks:
...
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "service-manual-publisher_production"
+    temppath: "/tmp/service-manual-publisher_production"
+    url: "govuk-integration-database-backups"
+    path: "postgresql-backend"
+  "pull_publishing_api_training_daily":
+    ensure: "present"
+    hour: "3"
```
Comment (schmie) [inline hieradata_aws/class/training/db_admin.yaml]: This is a big one - will likely take more than 1h to complete - maybe go for 5:00, here...

File: hieradata_aws/class/training/db_admin.yaml [hunk 2]
```diff
@@ -43,3 +43,201 @@ govuk_env_sync::tasks:
...
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "content_tagger_production"
+    temppath: "/tmp/content_tagger_production"
+    url: "govuk-integration-database-backups"
+    path: "postgresql-backend"
+  "pull_email_alert_api_training_daily":
+    ensure: "present"
+    hour: "2"
```
Comment (schmie) [inline hieradata_aws/class/training/db_admin.yaml]: Maybe give this one some more time as well.. ~3:30

File: hieradata_aws/class/training/db_admin.yaml [hunk 3]
```diff
@@ -43,3 +43,201 @@ govuk_env_sync::tasks:
...
+    action: "pull"
+    dbms: "mysql"
+    storagebackend: "s3"
+    database: "signon_production"
+    temppath: "/tmp/signon_production"
+    url: "govuk-integration-database-backups"
+    path: "mysql-backend"
+  "pull_whitehall_training_daily":
+    ensure: "present"
+    hour: "2"
```
Comment (schmie) [inline hieradata_aws/class/training/db_admin.yaml]: Let's go for 3:45 for this : )

Comment (camdesgov) [reply hieradata_aws/class/training/db_admin.yaml]: Good idea, changes made.

**Pull request merged**